django/contrib/sessions/backends/signed_cookies.py

```diff
@@ -17,7 +17,9 @@ def load(self):
                 # This doesn't handle non-default expiry dates, see #19201
                 max_age=settings.SESSION_COOKIE_AGE,
                 salt='django.contrib.sessions.backends.signed_cookies')
-        except (signing.BadSignature, ValueError):
+        except Exception:
+            # BadSignature, ValueError, or unpickling exceptions. If any of
+            # these happen, reset the session.
```

Comment:
I couldn't find a valid way of making the `Exception` catching cloak possible misconfiguration but I wonder if it wouldn't be better to make session serializers re-raise a common `DeserializationError` and catch it here.